sources/updater/waves/README.md

```diff
@@ -0,0 +1,39 @@
+# Update waves
+
+As mentioned in the [Bottlerocket update overview](../README.md), OS updates can include "waves" for staggered deployment.
+These waves are defined in the `manifest.json`, which lives in the TUF repository.
+Each time an OS update is made available, `manifest.json` is updated with the information pertinant to that update using [updata](../updog) or its related libraries.
+Waves may be supplied to `updata` on the command line using arguments, or passed as a TOML-formatted file.
+
+This directory contains a few examples of these update wave files.
+Specific details are encapsulated in each file, but they are:
+
+* `default-waves.toml`: A "normal" (non-emergency) deployment
```

Comment:
```suggestion
* `default-waves.toml`: A "normal" deployment
```

Only putting "non-emergency" on this entry implies that accelerated-waves is an emergency, which isn't exactly true.